packages/mocks/package.json

```diff
@@ -14,25 +13,21 @@
   },
   "author": "modern-web",
   "homepage": "https://github.com/modernweb-dev/web/tree/master/packages/mocks",
+  "type": "module",
   "exports": {
+    ".": {
+      "node": "./node.js",
```

Comment:
This is a bit unfortunate because we already have some projects that moved over to the new mocking solution.. Ill try to think if there's a nice way we can add support for node without breaking the existing imports